src/common/models/Projection.ts

```diff
@@ -521,10 +533,10 @@ export class Projection {
       ] as MetricsTimeseriesRow;
       actualsTimeseries.push(actualsTimeseriesrowForDate || null);
       metricsTimeseries.push(metricsTimeseriesRowForDate || null);
-      dates.push(currDate.toDate());
+      dates.push(new Date(currDate.getTime()));
 
       // increment the date by one
-      currDate = currDate.clone().add(1, 'days');
+      currDate.setUTCDate(currDate.getUTCDate() + 1);
     }
 
     // only keep futureDaysToInclude days ahead of today
```

Comment:
If you feel like doing some cleanup, we don't actually need `futureDaysToInclude` anymore, so you can remove that parameter and delete this code.